src/components/themes/blueradiance/theme.css

```diff
@@ -49,7 +49,7 @@ html {
 }
 
 .backgroundContainer.withBackdrop {
-    opacity: .93
+    opacity: .60
```

Comment:
why?

Reply:
more visible for backdrop

Reply:
Before
![ba](https://user-images.githubusercontent.com/32230989/69553887-f7d85780-0fb1-11ea-8027-f6cb62510e9e.JPG)

After
![tr](https://user-images.githubusercontent.com/32230989/69553930-09b9fa80-0fb2-11ea-8d0e-0367869446dc.JPG)

Reply:
Not a fan of this change, I think it should only go in the Blue Radiance theme.

Reply:
why backdrop option needed then?

Reply:
It actually looks a lot different on my machine, I'm not sure why yours is so dark by default.

Reply:
the opacity allmost 93%

Reply:
https://github.com/jellyfin/jellyfin-web/pull/594/commits/81b78035c060ea72b63224a4d44bee12e092f454